lua/lspconfig/server_configurations/drools_lsp.lua

```diff
@@ -0,0 +1,86 @@
+local util = require 'lspconfig.util'
+
+local get = function(config, section, key)
+  return config.settings and vim.tbl_get(config.settings, section, key) or nil
```

Comment:
```suggestion
  return vim.tbl_get(config.settings, section, key) and config.settings[section][key]  or nil 
```

Reply:
@glepnir Shortened the call to `vim.tbl_get` as suggested. Took me a minute 'cause I wanted to re-test everything. :)

Reply:
you can't only return `vim.tbl_get`  when find the key it return ture otherwise return nil.

Reply:
@glepnir Feeling stupid right about now, but changed again. Thanks for your patience, and apologies.